modules/audio/inference/direction_detection.cc

```diff
@@ -69,12 +69,12 @@ double DirectionDetection::EstimateDirection(
 
   int best_guess = 0;
   if (fabs(theta0) < fabs(theta1)) {
-    best_guess = theta1 > 0 ? (theta0 + 360) % 360 : (180 - theta0);
+    best_guess = theta1 > 0 ? fmod(theta0 + 360, 360) : (180 - theta0);
```

Comment:
Please use namespace specifier here: `std::fmod` rather than bare `fmod`.

Reply:
Got it

Reply:
Fixed